docs/DeveloperGuide.adoc

```diff
@@ -150,11 +166,366 @@ The `Storage` component,
 Classes used by multiple components are in the `seedu.addressbook.commons` package.
 
 == Implementation
```

Comment:
Probably not necessary to add sequence diagrams for every single command if they're similar